src/crawlee/basic_crawler/basic_crawler.py

```diff
@@ -679,6 +702,7 @@ async def __run_task_function(self) -> None:
             send_request=self._prepare_send_request_function(session, proxy_info),
             add_requests=result.add_requests,
             push_data=self._push_data,
+            log=self._logger,
```

Comment:
@vdusek not sure about this - it may be nice to make a new logger with a name like `crawlee.request_handler`, but that doesn't match any module, so we'd break a convention.

Reply:
it will log request queue / request list otherwise - am I correct? if so, I'd not override it

Reply:
more like `crawlee.basic_crawler.basic_crawler`, depending on your crawler